recordset/recordset.html

```diff
@@ -32,6 +32,17 @@ <h2 id="page-title">Records for {{ vm.tableDisplayName }}</h2>
                 <button class="btn btn-sm btn-default dropdown-toggle" type="button" data-toggle="dropdown">Items per page
                     <span class="caret"></span></button>
                 <ul class="dropdown-menu">
+                    <li ng-if="pageInfo.pageLimit !== 10 &&
+                        pageInfo.pageLimit !== 10 &&
+                        pageInfo.pageLimit !== 25 &&
+                        pageInfo.pageLimit !== 50 &&
+                        pageInfo.pageLimit !== 75 &&
+                        pageInfo.pageLimit !== 100 &&
+                        pageInfo.pageLimit !== 200"><a id="custom-page-size" href ng-click="pageLimit(pageInfo.pageLimit)">
+                        <span ng-show="true" class="glyphicon glyphicon-ok"></span>
+                        <span ng-show="false" class="glyphicon glyphicon-invisible"></span>
```

Comment:
Each place that you define 2 elements that work as an if/else block, it looks like the only difference is a class that you are applying. You could shrink these elements down to one element that looks like this:
```
<span class="glyphicon" ng-class="{'glyphicon-ok': true, 'glyphicon-invisible': false></span>
```